boards/nrf51dk/README.md

```diff
@@ -37,28 +61,35 @@ using `JLinkExe`.
 
 ## Debugging
 
-To manually connect to the board with a jlink programmer:
+Because the nRF51DK has integrated JTAG support, you can
+program it directly using gdb. In this setup, gdb connects
+to a process that gives access to the device over JTAG.
+First, create a `.gdbinit` file in the directory you are
```

Comment:
Why not just explain the commands that GDB needs to run, and people can put that into a .gdbinit file if they want to? Specifically, these instructions are actually not sufficient. The user would _also_ need to modify their `~/.gdbinit` file to include a line like: `set auto-load safe-path /home/[username]/tock/boards/nrf51dk`

Alternatively, there is a gdbinit file in the `jtag` subfolder which you can include with `-x` (I believe you wrote that file)